examples/seismic/tti/tti_example.py

```diff
@@ -17,15 +18,16 @@ def tti_setup(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     # Source and receiver geometries
     geometry = setup_geometry(model, tn)
 
-    return AnisotropicWaveSolver(model, geometry, space_order=space_order)
+    return AnisotropicWaveSolver(model, geometry, space_order=space_order,
+                                 kernel=kernel, **kwargs)
```

Comment:
It's not used in the init of the solver so no kernel shouldn't be there

Reply:
I thought your previous comment meant that I should add it here (?).
Which one is the signature?

Reply:
I meant need to be in the signature of `tti_setup` so that doesn't end up in kwargs